packages/rainbowkit/src/components/ConnectOptions/DesktopOptions.tsx

```diff
@@ -79,6 +80,7 @@ export function DesktopOptions({ onClose }: { onClose: () => void }) {
       setSelectedWallet(wallet);
       changeWalletStep(WalletStep.Connect);
     }
+    if (wallet.id === 'torus') onClose();
```

Comment:
What's happening here? 

Where possible, we would like to remove any "wallet-specific" code. We realize this might not always be possible, but we don't want to create separate branches of logic if we don't have to.

Reply:
@estrattonbailey just showed me that `onClose()` is being called here so that the RainbowKit modal will close, because the Torus web3auth modal is now behind the RainbowKit modal.

Reply:
i find another way more standard to handle this problem